scripts/lib/ipmi/ipmi_sim_lancontrol

```diff
@@ -32,7 +32,9 @@ link_mac() {
 }
 
 link_subnet() {
-    ifconfig $1 | sed -n -e 's/.*Mask:\([.0-9]*\).*/\1/p'
+    addr=$(ip -o -4 addr list $1 | awk '{print $4}')
```

Comment:
I guess you want to insert a `| head -n 1` before the `awk` call. Otherwise the function errors in case the device has multiple addresses.

Reply:
Done